syft/frameworks/keras/model/sequential.py

```diff
@@ -79,58 +75,35 @@ def step_fn():
 
 
 def shutdown(model):
-    subprocesses = getattr(model, '_subprocess_calls', None)
-    if subprocesses is None:
-        return
-
-    for subprocess in subprocesses.values():
-        subprocess.kill()
-        subprocess.communicate()
-
-
-
-_protocol_map = defaultdict(
-    tfe.get_protocol,
-    {'securenn': tfe.protocol.SecureNN, 'pond': tfe.protocol.Pond},
-)
-
-
-def _sanitize_share_argspec(workers, prot):
-    if not workers and prot is None:
-        raise RuntimeError("Invalid arguments supplied to model.share(): "
-                           "must supply TensorflowServerWorkers or a protocol object that corresponds")
-
-    if workers:
-        if len(workers) != 3:
-            raise RuntimeError("TF Encrypted expects three parties for its sharing protocols.")
-
-        players = OrderedDict([('server{}'.format(i), workers[i].host) for i in range(len(workers))])
-        config = tfe.RemoteConfig(players)
-        config.save('/tmp/tfe.config')
-        tfe.set_config(config)
-
-
-    if prot is None:
-        # Default is SecureNN
-        prot = 'securenn'
-    if isinstance(prot, str):
-        # here lies the dumbest, sneakiest bug of my life, so far...
-        # if you move this constructor call into _protocol_map,
-        # the protocol objects will be initialized before set_config above is called,
-        # which means any direct calls to prot will use the default device names
-        # (which come from LocalConfig). so we return the class from _protocol_map
-        # and instantiate it here
-        prot = _protocol_map[prot]()
-    return prot
-
-
-def _launch_tfserver_subprocesses(model):
-    subprocess_calls = {}
-    cmd = "python -m tf_encrypted.player --config /tmp/tfe.config {}"
-    hostmap = tfe.get_config().hostmap
-    for player_name in hostmap.keys():
-        subprocess_calls[player_name] = subprocess.Popen(cmd.format(player_name).split(' '))
-    setattr(model, '_subprocess_calls', subprocess_calls)
+    for worker in model._workers:
```

Comment:
Do we want call it `shutdown` or `workers_shutdown`? Because the workers launched is abstracted in share, it's not very clear what's the intent of shutdown.

Reply:
yup, `shutdown_workers()` seems appropriate

Reply:
not sure, by the same logic we could also have used `share_to_workers` and `serve_on_workers` yet `workers` seem implicit in those actions

Reply:
@iamtrask what's your take on this naming? what fits best with the terminology you guys are using elsewhere?